CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian-labs/db-replica/compare/release-2.1.4...master
 
+### Added
+- Add `ReplicaFailureStrategy`
```

Comment:
It would be good to explain what does it mean FailureStrategy ? 
is it circuit breaker? fallback ? what kind of failure? any failure?

Reply:
Replica failure = DualConnection fails to get a replica connection.

Reply:
Perhaps it's worth to add this to docs, there.

Reply:
And thanks for explaining, this is different than  db queries failing while using a replica connection , that would be more a CircuitBreaker  IIUC